clients/http/utils/common.go

```diff
@@ -54,7 +56,15 @@ func getBody(resp *http.Response) ([]byte, errors.EdgeX) {
 }
 
 // Helper method to make the request and return the response
-func makeRequest(req *http.Request) (*http.Response, errors.EdgeX) {
+func makeRequest(req *http.Request, authInjector interfaces.AuthenticationInjector) (*http.Response, errors.EdgeX) {
+	if authInjector != nil {
+		if err := authInjector.AddAuthenticationData(req); err != nil {
+			return nil, errors.NewCommonEdgeXWrapper(err)
+		}
+	} else {
+		fmt.Printf(" ** FIXME - REMOVE: No auth provider supplied")
```

Comment:
This can be removed because you provide injector during client initialization in https://github.com/edgexfoundry/go-mod-bootstrap/pull/404

Reply:
@hahattan Good catch!  Yes, it served its purpose and it no longer required.  It has been removed.